translations/pl.ts

```diff
@@ -644,7 +644,7 @@
     <message>
         <source>Map download error</source>
         <extracomment>notification summary</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>podsumowanie powiadomienia</translation>
```

Comment:
"Map download error" is the source :-) "notification summary" is comment for translators... https://github.com/Karry/osmscout-sailfish/blob/master/qml/pages/MapPage.qml#L89

Reply:
sorry

Reply:
fixed

Reply:
no problem :-)